common/history/countries/0_dvg_europe.txt

```diff
@@ -956,15 +956,25 @@
 	
 	c:GEN = { #Genoa
 		effect_starting_technology_tier_2_tech = yes
+		add_technology_researched = central_banking
 		
 		effect_starting_politics_conservative = yes
 		activate_law = law_type:law_presidential_republic
-		activate_law = law_type:law_landed_voting
-		activate_law = law_type:law_women_own_property
+		activate_law = law_type:law_autocracy
 		activate_law = law_type:law_national_supremacy
+		activate_law = law_type:law_women_own_property
+		activate_law = law_type:law_slave_trade
+		activate_law = law_type:law_colonial_exploitation
 		
 		set_ruling_interest_groups = {
-			ig_landowners
+			ig_armed_forces
+		}
+		ig:ig_landowners = {
+			set_interest_group_name = ig_genoese_patricians
+		}
```

Comment:
Maybe also add these:
			remove_ideology = ideology_paternalistic
			add_ideology = ideology_republican_paternalistic
			
Now land owners will support the monarchy.